frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -18,12 +21,7 @@ const CampCreationPage = (): React.ReactElement => {
   const [scheduleSessionsDummyTwo, setScheduleSessionsDummyTwo] = useState(
     false,
   );
-  const [registrationFormDummyOne, setRegistrationFormDummyOne] = useState(
-    false,
-  );
-  const [registrationFormDummyTwo, setRegistrationFormDummyTwo] = useState(
-    false,
-  );
+  const [visitedRegistrationPage, setVisitedRegistrationPage] = useState(false);
```

Comment:
This is for the stepper tab, right now I have that the registration form page gets marked as "completed" as soon as an admin visits it once, since an admin may choose to add no questions at all. Thoughts?